### DISCUSSION TITLE
Fade in/out anim parameters for round transitions.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1824] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Comercial games like CvS/CvS2, CFJ and KoF have this feature which is already available for screenpacks menus but not for rounds for some reason.

### Describe the solution you'd like

Fade in/out anim parameters for lifebars which allows you to set animations during round transitions.

### Additional context

I'd like to implement this animation in my own build.

https://youtu.be/PZ4nropdgsA?t=279

**Comment by potsmugen** [comment, marked as answer]
I think it's already implemented.
